libnavui-maps/src/main/java/com/mapbox/navigation/ui/maps/puck/LocationPuckOptions.kt

```diff
@@ -212,7 +212,7 @@ class LocationPuckOptions private constructor(
             fun navigationPuck(context: Context): LocationPuck = LocationPuck2D(
                 bearingImage = ContextCompat.getDrawable(
                     context,
-                    R.drawable.mapbox_navigation_puck_icon,
+                    R.drawable.mapbox_navigation_puck_icon2,
```

Comment:
Can we use a more appropriate name? Like `mapbox_navigation_puck_icon_themed` or sth like that?

Reply:
Why? `mapbox_navigation_puck_icon2` says it all. Version 2 of `mapbox_navigation_puck_icon`.

Reply:
Because later we'll have `mapbox_navigation_puck_icon3`, `mapbox_navigation_puck_icon4`, `mapbox_navigation_puck_icon10`. How will we know which one stands for what? While "themed" describes it: the one that supports different themes.
But you've worked with UI and resources on the project much more that I did so no strong objections here, just a suggestion.

Reply:
Do we need the old drawable at all? Can we simply update mapbox_navigation_puck_icon instead of adding a new drawable?

Reply:
@dzinad We'll address this later if needed. `mapbox_navigation_puck_icon2` is a good enough name for this asset.

> Do we need the old drawable at all? Can we simply update mapbox_navigation_puck_icon instead of adding a new drawable?
@Zayankovsky We... no. Customers that already integrated Drop-In UI... yes. To allow those customers to switch back to the old puck easily, we should keep it. Besides, the asset is a vector graphic that weighs only 926 bytes.